src/server/lib/notices/register-10dlc-brand.ts

```diff
@@ -66,42 +60,60 @@ export const get10DlcBrandNotices: OrgLevelNotificationGetter = async (
     messaging_service_sid: string;
     roles: string[];
   }[] = await query;
-  const notifications: (
-    | Register10DlcBrandNotice
-    | undefined
-  )[] = await Promise.all(
-    profiles.map(async ({ messaging_service_sid, roles }) => {
+  const ownedProfiles = profiles.filter(({ roles }) => roles.includes("OWNER"));
+
+  const registeredProfiles: Array<boolean> = await Promise.all(
+    ownedProfiles.map(async ({ messaging_service_sid }) => {
       const payload = {
-        operationName: "AnonGetTcr10DlcSurvey",
-        query: graphqlQuery,
+        operationName: "GetBillingAccountBySwitchboardProfileId",
+        query: fetchBillingAccountQuery,
         variables: {
           switchboardProfileId: messaging_service_sid
         }
       };
-      const response = await request
-        .post(`https://portal-api.spokerewired.com/graphql`)
+
+      const billingAccountResponse = await request
+        .post(PORTAL_API_URL)
         .send(payload);
 
-      if (!response.body.data.survey) {
-        return {
-          __typename: "Register10DlcBrandNotice",
-          id: messaging_service_sid,
-          tcrBrandRegistrationUrl: roles.includes("OWNER")
-            ? `https://portal.spokerewired.com/10dlc-registration/${messaging_service_sid}`
-            : null
+      if (billingAccountResponse.body?.data?.billingAccount?.id) {
+        const brandPayload = {
+          operationName: "AnonGetTcr10DlcBrandByBillingAccountId",
+          query: fetchBrandQuery,
+          variables: {
+            billingAccountId: billingAccountResponse.body.data.billingAccount.id
+          }
         };
+
+        const brandResponse = await request
+          .post(PORTAL_API_URL)
+          .send(brandPayload);
+
+        if (brandResponse.body.data?.brand?.campaigns?.nodes) {
+          return brandResponse.body.data?.brand?.campaigns?.nodes.some(
+            ({ state }: { state: string }) => state === "REGISTERED"
+          );
+        }
       }
-      return undefined;
+
+      return false;
     })
   );
 
-  const result = notifications.reduce<Register10DlcBrandNotice[]>(
-    (acc, notification) =>
-      notification !== undefined ? [...acc, notification] : acc,
-    []
-  );
+  console.log(registeredProfiles);
+
+  if (registeredProfiles.some((registered) => !registered)) {
```

Comment:
Question: Does this display the warning if there's multiple `ownedProfiles` but 1 is registered?